### PR TITLE
The Phazon is now salvageable!

### PR DESCRIPTION
**Pull request by alxmra**
## About The Pull Request

The Phazon is now salvageable and there is a small chance of dropping an anomaly core.

## Why It's Good For The Game

You can finally try to get some Phazon parts from a Phazon wreckage, and it may have a gift! Yes, there is a small chance of getting an anomaly core from the wreckage!

## Changelog
:cl:
add: Phazon is now salvageable
/:cl:

### CODE REVIEW

**Comment by JenqaDev** [comment]
this is one of my first pull requests, leave any tips and your opinion in the comments please

**Comment by PKPenguin321** [comment]
I assume the anomaly core will be the same type as the one used to build it?

**Comment by JenqaDev** [comment]
> I assume the anomaly core will be the same type as the one used to build it?

It will drop the normal anomaly core(called "anomaly core"). I may change it in the future, adding more rare items to the mechs' welder_salvage in a future PR.

**Comment by nemvar** [comment]
I just refactored these things. Please use the format used in #45904. You are using an outdated file.

**Comment by JenqaDev** [comment]
And do I keep the code I added to phazon_wreckage? Do I have to remove anomaly core from parts?

File: code/game/mecha/mecha_wreckage.dm [hunk 1]
```diff
@@ -249,6 +249,22 @@
...
+	. = ..()
+	var/list/parts = list(
+								/obj/item/mecha_parts/part/phazon_torso,
+								/obj/item/mecha_parts/part/phazon_head,
+								/obj/item/mecha_parts/part/phazon_left_arm,
+								/obj/item/mecha_parts/part/phazon_right_arm,
+								/obj/item/mecha_parts/part/phazon_left_leg,
+								/obj/item/mecha_parts/part/phazon_right_leg,
+								/obj/item/assembly/signaler/anomaly)
+	for(var/i = 0; i < 2; i++)
```
Comment (Akrilla) [inline code/game/mecha/mecha_wreckage.dm]: Use `pick_n_take` for this sort of thing

**Comment by AnturK** [comment]
Still needs update to current format. You just need parts from quick glance.

**Comment by AnturK** [comment]
@JenqaDev Needs update to current format.

**Pull request closed without merging**